controllers/csm_controller.go

```diff
@@ -697,6 +697,9 @@ func (r *ContainerStorageModuleReconciler) SyncCSM(ctx context.Context, cr csmv1
 	authorizationEnabled, _ := utils.IsModuleEnabled(ctx, cr, csmv1.AuthorizationServer)
 	if authorizationEnabled {
 		log.Infow("Create/Update authorization")
+		if err := r.reconcileAuthorizationCRDS(ctx, operatorConfig, cr, ctrlClient); err != nil {
+			return fmt.Errorf("failed to deploy authorization proxy server: %v", err)
```

Comment:
This error should be updated.

Reply:
I believe tests rely on this exact string. And it returns the detailed message from `reconcileAuthorizationCRDS`:
`return fmt.Errorf("unable to reconcile Authorization CRDs: %v", err)`

Or we can add another message here and, in this case, modify the tests